cert.py

```diff
@@ -35,12 +39,12 @@ def cert():
 @click.option('--key-digest', metavar='DIGEST', default='sha256', show_default=True, help='The digest to use when signing the request with its key (must be supported by openssl)')
 @click.option('--min-valid-time', type=argtypes.TimespanType, metavar='TIMESPAN', default='1d', show_default=True, help='If a certificate is found and its expiration lies inside of this timespan, it will be automatically requested and overwritten; otherwise no request will be made. The format for this option is "1d" for one day. Supported units are hours, days and weeks.')
 @click.option('--force', is_flag=True, default=False, show_default=True, help='Whether to force a request to be made, even if a valid certificate is found')
-@click.argument('domainroots', 'DOMAIN[:WEBROOT]', type=argtypes.DomainWebrootType, metavar='DOMAIN[:WEBROOT]', nargs=-1, required=True)
+@click.argument('domainroots', '[USER@HOST[@PORT]:]DOMAIN[:WEBROOT]', type=argtypes.RemoteDomainWebrootType, metavar='[USER@HOST[@PORT]:]DOMAIN[:WEBROOT]', nargs=-1, required=True)
```

Comment:
Wouldn't `DOMAIN:[[[USER@]HOST[:PORT]:]WEBROOT]` make slightly more sense, when considering multiple domains on the same webroot?
This way we don't have to change `DomainWebrootType` (besides changing the `.split()` to `.split(..., 1)`) and can handle the difference between local and remote solely inside `_store_webroot_validation()`.